tests/tests/lib.rs

```diff
@@ -6,6 +6,11 @@ mod blob;
 mod blocks;
```

Comment:
Why do we need changes in this file?

Reply:
We do not. A mistake with formatting. Fixed in e3d812b4c